apis/Google.Cloud.Diagnostics.AspNet/Google.Cloud.Diagnostics.AspNet/Trace/Labels.cs

```diff
@@ -27,6 +28,16 @@ namespace Google.Cloud.Diagnostics.AspNet
     /// </summary>
     internal static class Labels
     {
+        /// <summary>The version of the trace agent.</summary>
+        internal static readonly string AgentVersion = typeof(Labels)
+            .GetTypeInfo()
+            .Assembly
+            .GetCustomAttribute<AssemblyInformationalVersionAttribute>()
+            .InformationalVersion;
```

Comment:
I don't think so - so much of ASP.NET tends to rely on reflection that I'd be very surprised if we had problems.